### PR TITLE
Download zip files methods return array buffer

### PR DESCRIPTION
**Pull request by PavelLoparev**
Closes https://github.com/Smartling/api-sdk-nodejs/pull/115

Valid zip file can be saved like this:
```
fs.writeFileSync(
    "test.zip",
    Buffer.from(
        await apiClient.downloadFileAllTranslations(
            "<projected>",
            "<fileUri>",
            new DownloadFileParameters().setRetrievalType(RetrievalType.PUBLISHED)
        )
    )
);
```

### CODE REVIEW

File: api/base/index.ts [hunk 1]
```diff
@@ -89,7 +90,7 @@ export class SmartlingBaseApi {
         verb: string,
         uri: string,
         payload: Record<string, unknown> | string | FormData = null,
-        returnRawResponseBody = false,
+        returnRawResponseBody: ResponseBodyType | boolean = false,
```
Comment (PavelLoparev) [inline api/base/index.ts:93]: bc with internal package

**Comment by folklorelabs** [comment]
Why did you make a new branch and duplicate the changes in my PR? https://github.com/Smartling/api-sdk-nodejs/pull/115
I'm happy that y'all are addressing the issue, but taking credit for my work is pretty disappointing.

**Review by vfisyuk-smartling — APPROVED** [review]

**Comment by PavelLoparev** [comment]
> Why did you make a new branch and duplicate the changes in my PR? #115 I'm happy that y'all are addressing the issue, but taking credit for my work is pretty disappointing.

No one takes your credit. It's just easier to do all the changes in this repo directly instead of cloning your fork and pushing commits to your branch so that those changes appear in your PR.

Anyway, done changes in your branch.

**Comment by PavelLoparev** [comment]
Closing in favour https://github.com/Smartling/api-sdk-nodejs/pull/115

**Pull request closed without merging**